_docs/tasks/security/mutual-tls.md

```diff
@@ -43,93 +50,99 @@ Citadel is up if the "AVAILABLE" column is 1.
-   < date: Thu, 11 May 2017 18:59:42 GMT
-   < x-envoy-upstream-service-time: 2
-   ...
+   $ kubectl exec $(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name}) -c istio-proxy -- curl https://httpbin:8000/headers -o /dev/null -s -w '%{http_code}\n'
+   000
+   command terminated with exit code 77
    ```
 
-The service name and port are defined [here](https://github.com/istio/istio/blob/master/samples/bookinfo/kube/bookinfo.yaml).
-
-Note that Istio uses [Kubernetes service accounts](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/)
-as service identity, which offers stronger security than service name
-(refer [here]({{home}}/docs/concepts/security/mutual-tls.html#identity) for more information).
-Thus the certificates used in Istio do not have service names, which is the information that `curl` needs to verify
-server identity. As a result, we use `curl` option `-k` to prevent the `curl` client from aborting when failing to
-find and verify the server name (i.e., productpage.ns.svc.cluster.local) in the certificate provided by the server.
-
-Please check [secure naming]({{home}}/docs/concepts/security/mutual-tls.html#workflow) for more information
-about how the client verifies the server's identity in Istio.
+1. However, request will success if client certificate is provided
+    ```command
+    $ kubectl exec $(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name}) -c istio-proxy -- curl https://httpbin:8000/headers -o /dev/null -s -w '%{http_code}\n' --key /etc/certs/key.pem --cert /etc/certs/cert-chain.pem --cacert /etc/certs/root-cert.pem -k'
+    200
+    ```
 
-What we are demonstrating and verifying above is that the server accepts the connection from the client. Try not giving the client `--key` and `--cert` and observe you are not allowed to connect and you do not get an HTTP 200.
+    > Istio uses [Kubernetes service accounts](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/) as service identity, which offers stronger security than service name (refer [here]({{home}}/docs/concepts/security/mutual-tls.html#identity) for more information). Thus the certificates used in Istio do not have service names, which is the information that `curl` needs to verify server identity. As a result, we use `curl` option `-k` to prevent the `curl` client from aborting when failing to find and verify the server name (i.e., httpbin.ns.svc.cluster.local) in the certificate provided by the server.
 
+1. Request from pod without sidecar. For this demo, let's install another `sleep` service without sidecar. To avoid name conflicts, we put it in different namespace.
+    ```command
+    $ kubectl create ns legacy
+    $ kubectl apply -f samples/sleep/sleep.yaml -n legacy
+    ```
+Wait after the pod status change to `Running`, issue the familiar `curl` command. Request should fail as the pod doesn't have sidecar to help initiating TLS communication.
```

Comment:
change -> changes
The request should
have a sidecar
initiating -> initiate

Reply:
Done